<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbTableFieldDialog</class>
 <widget class="QDialog" name="DbTableFieldDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>字段属性</string>
  </property>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ObjectTypeEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>字段类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>成员类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FieldTypeComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Bool</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SByte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Byte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Uint16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Uint32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Uint64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Double</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DateTime</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>String</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>表模板成员</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>字段名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>字段描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DesFieldLlineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CommentEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ResFieldCombox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SyncButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="根据表模板成员同步其他关联属性"/>
   </property>
   <property name="text">
    <string>同步</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>170</y>
     <width>411</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
    <item row="1" column="4">
     <widget class="QSpinBox" name="spinBox_lwidth"/>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="cb_rtCurve">
      <property name="text">
       <string>实时曲线</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="cb_hisCurve">
      <property name="text">
       <string>历史曲线</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QSpinBox" name="spinBox_order">
      <property name="minimum">
       <number>-999</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>线宽:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>排序:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="cb_hisFilter">
      <property name="text">
       <string>历史过滤</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="cb_hisTable">
      <property name="text">
       <string>历史表格</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="MasterKeyCheckBox">
      <property name="text">
       <string>主键</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="EnableNullCheckBox">
      <property name="text">
       <string>允许为空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;example:&lt;/p&gt;&lt;p&gt;1. #FF0000&lt;/p&gt;&lt;p&gt;2. rgba(255,255,00,1)&lt;/p&gt;&lt;p&gt;3. red&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>曲线颜色(#RGB)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ColorEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="extEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>扩展</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
